apps/styleguide/src/mixins/TableMetadataMixin.vue

```diff
@@ -1,3 +1,6 @@
+<template>
+  <div>Not meant for direct use</div>
```

Comment:
suggest to move it outside of example folder

Reply:
I like to have this for testing purposes. So replaces with useful template.